background/messages/mCreateWindow.ts

```diff
@@ -15,19 +15,75 @@ const handler: PlasmoMessaging.MessageHandler<
   RequestBody,
   ResponseBody
 > = async (req, res) => {
-  console.log(`[background][messaging][window] Request: ${JSON.stringify(req)}`)
-
-  await windows.create({
-    url: createWindowUrl,
-    focused: true,
-    type: "popup",
-    width: 385,
-    height: 720
-  })
+  console.log(
+    `[background][messaging][window] Request: ${JSON.stringify(req, null, 2)}`
+  )
+
+  const createWindowUrl = `${runtime.getURL(
+    "tabs/mCreateWindow.html"
+  )}?origin=${req.body.origin}&account=${req.body.account}&tabId=${
+    req.sender.tab.id
+  }`
+
+  const window = await createWindowAsync(createWindowUrl)
+  console.log(
+    `[background][messaging][window] window: ${JSON.stringify(window, null, 2)}`
+  )
+
+  //   const tab = await createTabAsync(createWindowUrl)
+  //   console.log(
+  //     `[background][messaging][window] tab: ${JSON.stringify(window, null, 2)}`
+  //   )
 
   res.send({
     out: `Opened: ${createWindowUrl}`
   })
 }
 
+function createWindowAsync(createWindowUrl: string) {
+  return new Promise(async (resolve, reject) => {
+    let createdWindow = null
+    try {
+      createdWindow = await windows.create({
+        url: createWindowUrl,
+        focused: true,
+        type: "popup",
+        width: 385,
+        height: 720
+      })
+    } catch (e) {
+      reject(e)
```

Comment:
Sorry I forgot the return expression 🥲 Same for the tab.

```suggestion
      reject(e)
      return
```